File: Programmers/피보나치 수.cpp
#include <string>
#include <vector>

using namespace std;
int arr[100002];

int solution(int n) {
    arr[1] = 1;
    for(int i = 2; i <= n; i++)
        arr[i] = (arr[i-1] + arr[i-2]) % 1234567;
    return arr[n];

}

File: Programmers/피보나치 수.java
class Solution {
	static int[] fibo = new int[100002];
    public int solution(int n) {
        fibo[1] = 1;
        for(int i = 2; i <= n; i++)
        	fibo[i] = (fibo[i - 1] + fibo[i - 2]) % 1234567;
        
        return fibo[n];
    }
}